cpp/oneapi/dal/algo/kmeans/backend/gpu/infer_kernel_dpc.cpp

```diff
@@ -154,23 +158,21 @@ struct infer_kernel_gpu<Float, method::lloyd_csr, task::clustering> {
                                                                          arr_centroids,
                                                                          arr_centroid_squares,
                                                                          { data_squares_event });
+
         auto assign_event = assign_clusters(queue,
-                                            values,
-                                            column_indices,
-                                            row_offsets,
+                                            row_count,
+                                            data_handle,
                                             arr_data_squares,
                                             arr_centroids,
                                             arr_centroid_squares,
                                             distances,
                                             arr_responses,
                                             arr_closest_distances,
                                             { data_squares_event, centroid_squares_event });
+
         auto objective_function =
             calc_objective_function(queue, arr_closest_distances, { assign_event });
-        {
-            // Reduce objective function value over all ranks
-            comm.allreduce(objective_function).wait();
```

Comment:
whats the reason of removing this comm operation?

Reply:
Distributed implementation for CSR method is incomplete and is not functional. Removing for now.